src/routes/static.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
```

Comment:
Is this file needed?

Reply:
Yes, the next router is imported from express
```
const express = require("express");
const router = express.Router();
```

Reply:
I mean, is `src/routes/static.js` needed? It seems like it just contains a dummy route for the root path.

Reply:
Oh, I see. No, this isn't needed. Removed.